tests/zquantum/core/measurement_test.py

```diff
@@ -223,28 +223,26 @@ def test_convert_bitstring_to_int():
     assert convert_bitstring_to_int(bitstring) == 42
 
 
-def test_check_parity_odd_string():
-    bitstring = "01001"
+def test_check_parity_of_strings():
```

Comment:
Since `check_parity` should stay, these should be added as new tests, not replace old ones.